src/parser_utils.c

```diff
@@ -102,6 +109,34 @@ t_err	ft_create_str_arr(t_cmd *tmp, int count_arg, int count_delim)
 	}
 	else
 		tmp->delims = NULL;
+	err = ft_create_str_arr2(tmp, count_out);
+	return (err);
+}
+
+/**
+ * @brief Reserve memory space for outfiles string array.
+ * 
+ * @param tmp 		New cmd struct to be created.
+ * @param count_out Amount of outfiles.
+ * @return t_err 	ERR_MALLOC, SUCCESS
+ */
+t_err	ft_create_str_arr2(t_cmd *tmp, int count_out)
```

Comment:
ft_create_str_arr_outfile(). Just to be a little bit more verbo~se

Reply:
Created extra malloc function